File: src/main/r/problems001_020/Problem1.R
solve <- function() {
    limit <- 1000
    multiples <- c(3, 5)
    values <- rep(seq(min(multiples), limit-1), length(multiples))
    return(sum(unique(values[(values %% sort(rep(multiples, length(values)/2))) == 0])))
}

File: src/main/python/problems001_020/Problem1.py
import numpy as np
import operator as op


def solve():
    limit = 1000
    multiples = [3, 5]
    r = np.arange(min(multiples), limit)

    return sum(r[reduce(op.or_, [np.mod(r, m) == 0 for m in multiples])])
